source/bdDetect.py

```diff
@@ -618,6 +648,9 @@ def addUsbDevices(self, type: DeviceType, ids: Set[str]):
 				f"Invalid IDs provided for driver {self._driver!r}, type {type!r}: "
 				f"{', '.join(malformedIds)}"
 			)
+		if useAsFallBack:
+			FallbackDevicesStore.fallBackDevices.update([(self._driver, type, id) for id in ids])

```

Comment:
@coderabbitai I have the feelling that this can be. It shouldn't have to be a list. Is that correct?
```suggestion
			FallbackDevicesStore.fallBackDevices.update(((self._driver, type, id) for id in ids))

```@coderabbitai Could you suggest how to improve this line?